doc/02-Installation.md

```diff
@@ -1,15 +1,24 @@
-# Installation
+<!-- {% if index %} -->
+# Installing Icinga Web Graphite Integration
 
-## Requirements
+The recommended way to install this Integration is to use prebuilt
```

Comment:
`integration` is not a name so write it lowercase please. Although, I would avoid this altogether and write just `It is recommended to use prebuilt...`

Reply:
Of course, in `Icinga Web Graphite Integration` it's part of the name and hence the first letter is uppercase. But used individually it's just wrong to do that.